test/setup/setup.js

```diff
@@ -5,7 +5,7 @@ module.exports = function() {
   Backbone.$ = jQuery;
   Backbone.ChildViewContainer = require('backbone.babysitter');
   Backbone.Radio = require('backbone.radio');
-  var Marionette = require('../../src/backbone.marionette');
+  var Marionette = require('../../src/backbone.marionette').default;
```

Comment:
You can get rid of this by using https://github.com/59naga/babel-plugin-add-module-exports
I was super annoying by this when moving to babel v6